cli/src/cmd/completion/zsh.go

```diff
@@ -0,0 +1,80 @@
+package completion
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"runtime"
+
+	execShell "cli/util/exec"
+	fileUtil "cli/util/file"
+
+	"github.com/luno/jettison/log"
+	"github.com/spf13/cobra"
+)
+
+func genZshCompletionCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "zsh",
+		Short: "Generate the autocompletion script for zsh",
+		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
The args are not being used in the function

Reply:
The `Run` property for type cobra.Command is of type `func(cmd *cobra.Command, args []string)`. So, although we don't use `args`, it is required for the property type.